packages/openapi3/test/tsp-openapi3/utils/tsp-for-openapi3.ts

```diff
@@ -24,7 +25,17 @@ export interface OpenAPI3Options {
   paths?: Record<string, OpenAPI3PathItem>;
 }
 
-export async function tspForOpenAPI3({ parameters, paths, schemas }: OpenAPI3Options) {
+export async function tspForOpenAPI3(props: OpenAPI3Options) {
```

Comment:
I also found I needed to change it in the symbol refactor PR

```ts
export async function tspForOpenAPI3WithProgram({ parameters, paths, schemas }: OpenAPI3Options) {
  const openApi3Doc: OpenAPI3Document = {
    info: {
      title: "Test Service",
      version: "1.0.0",
    },
    openapi: "3.0.0",
    paths: { ...paths },
    components: {
      schemas: {
        ...(schemas as any),
      },
      parameters: {
        ...(parameters as any),
      },
    },
  };

  const code = await convertOpenAPI3Document(openApi3Doc);
  const testableCode = wrapCodeInTest(code);
  const host = await createTestHost({
    libraries: [HttpTestLibrary, OpenAPITestLibrary, OpenAPI3TestLibrary],
  });
  host.addTypeSpecFile("main.tsp", testableCode);

  const { TestService } = await host.compile("main.tsp");

  assert(
    TestService?.kind === "Namespace",
    `Expected TestService to be a namespace, instead got ${TestService?.kind}`,
  );
  return [TestService, host.program] as const;
}
export async function tspForOpenAPI3(options: OpenAPI3Options) {
  return (await tspForOpenAPI3WithProgram(options))[0];
}
```

do you think we could have a result that also include the program.